environment.yml

```diff
@@ -2,14 +2,14 @@ name: vector
 channels:
   - conda-forge
 dependencies:
-  - jupyterlab >=1.2
+  - jupyterlab>=1.2
   - nb_conda_kernels
-  - pip >=18
+  - pip>=18
   - pytest
-  - root >=6.18.04
-  - uproot >=3.11
-  - numba >=0.46
+  - root>=6.18.04
+  - uproot>=3.11
+  - numba>=0.46
   - pip:
-    - "awkward1>=0.2.3,!=0.2.4"
+    - "awkward1>=0.2.3"
```

Comment:
Let's just make this 0.2.5 - there was a bad bug in 0.2.4 that broke vector, which is why it was removed here. Yes, we can use 0.2.3 currently, but there's no reason to include it.